docs/src/diagnostics/users_diagnostics.md

```diff
@@ -0,0 +1,58 @@
+# default diagnostics 

```

Comment:
This page has to include instruction how to add a diagnostic that is not default